README.md

```diff
@@ -13,83 +13,50 @@
-
-## 🎖 贡献者
-
-<a href="https://github.com/PaddlePaddle/PaddleOCR/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=PaddlePaddle/PaddleOCR&max=400&columns=20"  width="800"/>
-</a>
-
-## ⭐️ Star
-
-[![Star History Chart](https://api.star-history.com/svg?repos=PaddlePaddle/PaddleOCR&type=Date)](https://star-history.com/#PaddlePaddle/PaddleOCR&Date)
-
-## 📄 许可证书
-
-本项目的发布受 [Apache License Version 2.0](./LICENSE) 许可认证, 欢迎大家使用和贡献。
+# Optocycle PaddleOCR Fork
+
+This fork includes configurations for fine-tuning PaddleOCR on license plate data, along with an extension for MLflow logging.
+
+## Fine-tuning a model
+
+These instructions are based on the guide provided [here](https://paddlepaddle.github.io/PaddleOCR/latest/en/quick_start.html). 
+1. Install PaddlePaddle
+    - CPU installation: `python -m pip install paddlepaddle==3.0.0rc1 -i https://www.paddlepaddle.org.cn/packages/stable/cpu/`
+    - GPU installation: `python -m pip install paddlepaddle-gpu==3.0.0rc1 -i https://www.paddlepaddle.org.cn/packages/stable/cu118/`
+2. Open the cloned repository and download a pretrained model.
+```bash
+cd PaddleOCR/
+# Download the pre-trained model of en_PP-OCRv3 (the weights can be also found in MinIO)
+wget -P ./pretrain_models/ https://paddleocr.bj.bcebos.com/PP-OCRv3/english/en_PP-OCRv3_rec_train.tar
+# Decompress model parameters
+cd pretrain_models
+tar -xf en_PP-OCRv3_rec_train.tar && rm -rf en_PP-OCRv3_rec_train.tar
+cd ..
+```
+3. Ensure your dataset is formatted for PaddleOCR and move it to the `datasets/` directory: `mv /path/to/paddleocr_dataset ./datasets`
+4. Fine-tune the model:
+```bash
+python3 tools/train.py -c configs/rec/PP-OCRv3/slplates_finetuning.yml -o Global.pretrained_model=pretrain_models/en_PP-OCRv3_rec_train/best_accuracy
+```
+5. Export the fine-tuned model to an inference model and specify a custom destination directory (e.g., `./slplates_inference_model`):
+```bash
+python3 tools/export_model.py -c configs/rec/PP-OCRv3/slplates_finetuning.yml -o Global.pretrained_model=output/normal_finetuned_slplates_paddleocr/best_model/model  Global.save_inference_dir=./slplates_inference_model
+```
+
+## Enabling MLflow
+
+To enable MLflow logging during fine-tuning, add the following to your configuration file:
+```yaml
+mlflow:
+  mlflow_log_every_n_iter: <n_iter>  # Log training metrics every n iterations (evaluation metrics are always logged)
+  name: <name_of_the_run>  # Name of the MLflow run (e.g., "lp_finetuned")
+```
+Additionally, create a `.env` file by copying `.env.example` and filling in the correct values."
+
+## 📄 License
+
+This project is released under [Apache License Version 2.0](./LICENSE).
+
+### Changes
+In addition to the modifications described in the header of the original files, the following changes have been made:
+- The `setup.py` and `MANIFEST.in` files have been removed as the project has been switched to using Poetry for dependency management and packaging.
```

Comment:
It is not licensed under apache. We do not add license to this software at all since it is proprietary to us. The code that you used is licensed under apache